forum-integration/src/Listener/SynchronizeUser.php

```diff
@@ -101,7 +107,20 @@ public function synchronize(Events\Event $event)
             }
         }
     }
-
+    /**
+    * 判断论坛类型
+    * @return discuz|flarum
```

Comment:
```suggestion
    * @return string
```

因为这里应该写返回值的类型，而不是具体的值。